### PR TITLE
Add mtu variable

### PR DESCRIPTION
**Pull request by SchSeba**
This PR add a variable to configure the mtu on the veth interface.
This feature is to make the ovs cni equal to the bridge cni plugin.

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/ovs-cni/79)

### CODE REVIEW

File: pkg/plugin/plugin_test.go [hunk 1]
```diff
@@ -207,7 +216,19 @@ var _ = Describe("CNI Plugin", func() {
...
+				testAddDel(conf, false, false)
+			})
+		})
+		Context("with MTU set on port", func() {
+			conf := fmt.Sprintf(`{
+				"cniVersion": "0.3.1",
+				"name": "mynet",
+				"type": "ovs",
+				"bridge": "%s",
+                "mtu": %d
```
Comment (phoracek) [inline pkg/plugin/plugin_test.go]: please fix the indentation

File: pkg/plugin/plugin.go [hunk 2]
```diff
@@ -116,15 +117,15 @@ func generateRandomMac() net.HardwareAddr {
...
+func setupVeth(contNetns ns.NetNS, contIfaceName string, requestedMac string, mtu int) (*current.Interface, *current.Interface, error) {
 	hostIface := &current.Interface{}
 	contIface := &current.Interface{}
 
 	// Enter container network namespace and create veth pair inside. Doing
 	// this we will make sure that both ends of the veth pair will be removed
 	// when the container is gone.
 	err := contNetns.Do(func(hostNetns ns.NetNS) error {
-		hostVeth, containerVeth, err := ip.SetupVeth(contIfaceName, 1500, hostNetns)
+		hostVeth, containerVeth, err := ip.SetupVeth(contIfaceName, mtu, hostNetns)
```
Comment (phoracek) [inline pkg/plugin/plugin.go]: what if the MTU is not specified? do we set it to 0?

File: pkg/plugin/plugin_test.go (hunk 1 above)
Comment (SchSeba) [reply pkg/plugin/plugin_test.go]: done

File: pkg/plugin/plugin.go (hunk 2 above)
Comment (SchSeba) [reply pkg/plugin/plugin.go]: when it's 0 the kernel allocate a default value.

added a check for that

**Comment by phoracek** [comment]
Thanks @SchSeba!

**Pull request merged**